docs/hardware/receiver-selection.md

```diff
@@ -39,6 +42,7 @@ ExpressLRS is hashtag blessed with the benefit of there being many receivers ava
 | HappyModel ES900RX | :moneybag::moneybag: | :straight_ruler: | :thumbsup_tone3:  | :thumbsup_tone3:  | :whale::whale: | :thumbsup_tone3:  | :thumbsup_tone3:  |
 | Jumper R9 Mini | :moneybag::moneybag: | :straight_ruler::straight_ruler: | :thumbsup_tone3:  | :thumbsup_tone3:  | :whale::whale: | :thumbsdown_tone3: | :thumbsdown_tone3: |
 | NamimnoRC Voyager | :moneybag::moneybag::moneybag: | :straight_ruler::straight_ruler: | :thumbsup_tone3:  | :thumbsup_tone3:  | :whale::whale: | :thumbsdown_tone3: | :thumbsup_tone3:  |
+| GEPRC Nano | :moneybag::moneybag: | :straight_ruler::straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale::whale: | :thumbsup_tone3:  | :checkered_flag: |
```

Comment:
Isn't this a duplicated entry w. the Line 21?
Also there are two: NanoSE and Nano

| GEPRC NanoSE | :moneybag::moneybag: | :straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale: | :thumbsup_tone3:  | :checkered_flag: |
| GEPRC Nano | :moneybag::moneybag: | :straight_ruler::straight_ruler: | :thumbsdown_tone3: | :thumbsdown_tone3: | :whale::whale: | :thumbsup_tone3:  | :checkered_flag: |

Reply:
Internally, there's only a Nano, but I did thought of putting in one for the SE

this section is for the 900MHz ones